spec/helpers/item_seasons_helper_spec.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ItemSeasonsHelper, type: :helper do
+  describe 'randomized rotation' do
+    it 'adds a random rotation to a button' do
+      expect(helper.randomized_rotation).to include('rotate')
+    end
+  end
```

Comment:
**issue:** you are not testing much here. All items have rotate, thus you just know it return a member of the array. Testing randomness is tricky and probably a bit flaky. So I'd suggest to do something like this:

```rb
array1 = 10.times.map { helper.randomized_rotation }
array2 = 10.times.map { helper.randomized_rotation }
expect(array1).not_to match(array2)
```

Reply:
Yeah, I had a hard time trying to figure out how I could test this one. I thought I could at least test if the rotation was being added.